src/components/Reports/ReportsBox/ReportsBox.js

```diff
@@ -1,6 +1,16 @@
 import React from 'react';
 import { InjectAppServices } from '../../../services/pure-di';
 import { FormattedMessage, FormattedDate } from 'react-intl';
+import { addDays } from '../../../utils';
+
+const DateRangeText = ({ dateFrom, dateTo }) => {
+  return (
+    <>
+      <FormattedDate value={dateFrom} /> <FormattedMessage id="reports_box.to" />{' '}
+      <FormattedDate value={addDays(dateTo, -1)} />{' '}
```

Comment:
Why subsctract 1 here, instead of doing it outside? you could reuse in another scenario like today. Right?

Reply:
Else I would change the name of the component to avoid doing more than what the name stands for, like for example `DateRangeTextToPreviousDayOfEndDate` I guess....

Reply:
Yo dejaría un nombre genérico y cuando necesitemos más escenarios lo haría más inteligente para que los soporte.

Reply:
Ya hablamos sobre esto en una call y decidimos poner un nombre representativo y dejar la logica del lado del componente que se abstraiga la logica